src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion_img2img.py

```diff
@@ -66,6 +68,11 @@
 
 # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion_img2img.preprocess
 def preprocess(image):
+    warnings.warn(
+        "The function preprocess is deprecated and will be removed. Please"
+        " use VAEImageProcessor.preprocess instead.",
```

Comment:
```suggestion
        " use `processor = VAEImageProcessor()` and `processor.preprocess(image)` instead.",
```

Reply:
Actually sorry I didn't think this through. We need to copy this to all other processors but those don't have the processor yet, so let's maybe remove the warning message so that it doesn't show up for the other processors.